--- rsync.orig	2015-12-30 19:49:12.101247644 +0300
+++ rsync	2015-12-30 19:49:20.461247855 +0300
@@ -5,7 +5,7 @@
 #  Use "inetd" if you want to start the rsyncd from inetd,
 #  all this does is prevent the init.d script from printing a message
 #  about not starting rsyncd (you still need to modify inetd's config yourself).
-RSYNC_ENABLE=false
+RSYNC_ENABLE=true
 
 # which file should be used as the configuration file for rsync.
 # This file is used instead of the default /etc/rsyncd.conf
